cloudant-client/src/main/java/com/cloudant/client/internal/util/SelectorUtils.java

```diff
@@ -17,18 +17,89 @@
 import static com.cloudant.client.org.lightcouch.internal.CouchDbUtil.assertNotEmpty;
 import static com.cloudant.client.org.lightcouch.internal.CouchDbUtil.assertNotNull;
 
+import com.cloudant.client.api.query.Selector;
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParseException;
 
 public class SelectorUtils {
 
-    public static JsonObject getPartialFilterSelectorFromString(String selectorJson) {
-        return getSelectorFromString("partial_filter_selector", selectorJson);
+    // Selector object keys
```

Comment:
I wonder whether this has overlap with my `Helpers` class

Reply:
Moved `SelectorUtils` methods into `Helpers`, unfortunately it didn't look like I could re-use any of the methods. See b2c9a25a46f7754a1a681737a1694b5b0dd0c6eb